packages/client/src/net/peer/peer.ts

```diff
@@ -38,17 +37,22 @@ export interface PeerOptions {
  * Network peer
  * @memberof module:net/peer
  */
-export class Peer extends EventEmitter {
+export abstract class Peer extends EventEmitter {
```

Comment:
One general question here is what advantage do we have with an abstract `peer` class and then a concrete `RLPXPeer` subclass?  We don't have multiple network options so this feels like abstraction without a purpose.

Reply:
Not super sure here, I just made it abstract since we instantiate it nowhere and it had some obvious abstract methods which should be implemented (empty method which just throws if you call it).